crates/solver-discovery/src/implementations/offchain/_7683.rs

```diff
@@ -675,11 +733,37 @@ async fn handle_intent_submission(
 				.into_response();
 		}
 	};
+	tracing::info!("Parsed order");
+	// Log parsed order details per line for readability
+	let first_output = order.outputs.get(0).map(|o| {
+		(
+			o.oracle.0,
+			o.settler.0,
+			o.chainId,
+			o.token.0,
+			o.amount,
+			o.recipient.0,
+			o.call.len(),
+			o.context.len(),
+		)
+	});
+	let lines = format_standard_order_lines(
+		order.user,
+		order.nonce,
+		order.originChainId,
+		order.expires,
+		order.fillDeadline,
+		order.inputOracle,
+		&order.inputs,
+		first_output,
+	);
+	for line in lines { tracing::info!("{}", line); }
```

Comment:
This printing a new log for every line? Why don't we just construct a JSON object from the order and use {:#} to print it?